.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-  - 'node'
+  - '12'
+install: skip
+cache:
+  npm: false
```

Comment:
Looks like running `./build-package.sh` installs and build npm modules with chromium headers, for atom, electron, etc..

Later on `npm rebuild` is used before running additional tests in node, to build native modules with node headers instead.

I think these rebuilt modules can trip up future CI runs, as apm will attempt to use the native modules compiled for node. Then things go 💥 

This appears to work 👀